apps/scan/backend/scripts/copy_sheets.ts

```diff
@@ -0,0 +1,99 @@
+import fs from 'fs';
+import path from 'path';
+import { v4 as uuid } from 'uuid';
+import { AcceptedSheet } from '@votingworks/backend';
+import {
+  assert,
+  assertDefined,
+  extractErrorMessage,
+} from '@votingworks/basics';
+import { safeParseNumber } from '@votingworks/types';
+
+import { SCAN_WORKSPACE } from '../src/globals';
+import { Store } from '../src/store';
+
+const usageMessage = `Usage: copy-sheets <target-sheet-count>
+
+Arguments:
+  <target-sheet-count>\tThe target total sheet count after copying`;
+
+interface CopySheetsInput {
+  targetSheetCount: number;
+}
+
+function checkEnvironment(): void {
+  assert(process.env.NODE_ENV !== 'production', 'Cannot be run in production');
+}
+
+function parseCommandLineArgs(args: readonly string[]): CopySheetsInput {
+  if (args.length !== 1 || !safeParseNumber(args[0]).isOk()) {
+    console.log(usageMessage);
+    process.exit(0);
+  }
+  const targetSheetCount = safeParseNumber(args[0]).unsafeUnwrap();
+  return { targetSheetCount };
```

Comment:
Consider:
- using `safeParseInt` instead of `safeParseNumber`
- only parsing once
- using type narrowing instead of `unsafeUnwrap`
- printing the usage message to stderr instead of stdout
- exiting with a non-zero exit status so e.g. a scripting-based caller knows it failed
```suggestion
  const parseTargetSheetCountResult = safeParseInt(args[0]);
  if (parseTargetSheetCountResult.isErr()) {
    process.stderr.write(`${usageMessage}\n`);
    process.exit(1);
  }
  return { targetSheetCount: parseTargetSheetCountResult.ok() };
```

Reply:
👍 https://github.com/votingworks/vxsuite/pull/4225/commits/3a94d52cd7682d9c2b56425f0a71eddf789d5043